test/functional/shared/resources/testdata/recipes/test-bicep-recipes/rabbitmq-recipe.bicep

```diff
@@ -1,33 +1,32 @@
 import kubernetes as kubernetes {
   kubeConfig: ''
-  namespace: context.runtime.kubernetes.namespace
+  namespace: 'radius-testing'
 }
 
-param context object
 
 @description('Specifies the RabbitMQ username.')
 param username string = 'guest'
 
 @description('Specifies the RabbitMQ password.')
 @secure()
-param password string
+param password string = 'guest'
 
 resource rabbitmq 'apps/Deployment@v1' = {
   metadata: {
-    name: 'rabbitmq-${uniqueString(context.resource.id)}'
+    name: 'rabbitmq-test'
```

Comment:
Wouldn't this create a problem in the tests that are running in parallel? Like long-running test and a functional test? I believe we should always make sure to have unique resource names.